src/main/java/dk/bec/polonez/reservationsystem/controller/ReservationController.java

```diff
@@ -22,12 +24,17 @@ public ReservationController(ReservationService reservationService) {
     }
 
     @GetMapping
-    public List<ResponseReservationDto> getAllReservations() {
-        return reservationService.getAll(false);
+    public List<ResponseReservationDto> getAllReservations(@RequestParam boolean sortedByCreationDate) {
+        List<ResponseReservationDto> reservations = reservationService.getAll();
+
+        if(sortedByCreationDate && !reservations.isEmpty())
```

Comment:
**advice** IMO sorting is a part of "business logic" so it should be implemented in ```ReservationService``` (not in ```ReservationController```). Of your it's debatable, one can say that it's pure GUI-logic.